crates/workspace/src/pane.rs

```diff
@@ -152,6 +152,8 @@ actions!(
         SplitRight,
         SplitDown,
         TogglePreviewTab,
+        CopyPath,
```

Comment:
We already have `outline_panel::CopyRelativePath` and `project_panel::CopyRelativePath`, and this will add another relatively specific namespace, `pane::`, into the mix.

I would rather use a generic `workspace::` one for these new two (same as https://github.com/zed-industries/zed/pull/13976/files#diff-041673bbd1947a35d45945636c0055429dfc8b5985faf93f8a8a960c9ad31e28 does) 

So that later we could change both panels to use that `workspace::CopyPath` and `worksace::CopyRelativePath` pair, and future places like editor can reuse the same `workspace::` action too.

Reply:
Okay, I will add those two to the workspace crate

Reply:
Done!